psycopg/psycopg/_copy_base.py

```diff
@@ -216,10 +216,11 @@ def __init__(self, transformer: Transformer, encoding: str = "utf-8"):
         self._encoding = encoding
 
     def parse_row(self, data: Buffer) -> tuple[Any, ...] | None:
+        rv: tuple[Any, ...] | None = None
```

Comment:
can it be just `tuple`?

Reply:
No, I don't think it's a valid type annotation.